src/nxdoc/nuxeo-add-ons/amazon-s3-online-storage.md

```diff
@@ -398,6 +400,10 @@ Here is a sample AWS S3 Policy that you can use; make sure that you replace&nbsp
 }
 ```
 
+### AWS S3 Cleanup Lifecycle Rule
+
+If versioning is enabled on your s3 bucket, you should define a Cleanup Lifecycle rule to remove expired object delete markers. As a matter of fact, in the case of s3 versioning enabled, the [Orphaned Blobs GC]({{page page='garbage-collecting-orphaned-blobs'}}) will only add a delete marker on the garbage-collected object. This object will be permanently deleted only if such a lifecycle rule is defined.
```

Comment:
I think it deserves a `{{#> callout type='warning' }}` section,
may be add a reference link on what is "S3 versioning" and lifecycle rules.